src/pages/debug-app/debug-app.tsx

```diff
@@ -744,7 +741,7 @@ export const DebugApp = () => {
         getColor: (d) => d.color,
         getWidth: 2,
       }),
-      // @ts-expect-error
+      // @ts-expect-error - Expected 0 arguments, but got 1.
```

Comment:
Add the constructor to the class
```
constructor(public props: any) {
    super(props);
  }
```

Reply:
Unfortunately it doesn't help I got another error because of this constructor:
![image](https://user-images.githubusercontent.com/70207219/153424617-7d280b59-aeb3-469a-b0c5-dfb44c5d62e7.png)